manuscript/04.discussion.md

```diff
@@ -1,11 +1,51 @@
+2. It is important to take heritability and cell-cell relations into account while studying the evolution and state transitions.
 
-Naturally, the degree of difference between states that are labeled as "distinct" plays an important role in deciphering which state each cell would belong to. Hence, the model is dependent on some measure of distance between the existing states and may perform poorly when the states are close. The tHMM accurately performs maximum likelihood estimation using cell observations from each lineage in the population. Thus, the improved performance accuracy and decrease in its variance as lineage number increases validates the model architecture. Parameter estimation and initial and transition probability matrix estimations were accurate in almost all cases of synthetic data and the tHMM was able to better distinguish the separate subpopulations as more lineages were added. Furthermore, the model operates equally well whether the population of interest owns a pre-existing phenotypic heterogeneity or the cells acquire diverse phenotypes as a result of drug treatment. The model selection validated by the BIC metric asserted the reliability of tHMM in predicting the most likely number of states within a population (Fig. @fig:sBIC).
+3. Previous studies have investigated identifying phenotypic heterogeneity using models.
 
-As a follow up study, We intend to design assays to experimentally validate the number of states predicted by tHMM, and study the extent and sources of differences between the identified states. The current version of this pipeline is most accurate when populations consist of at least 100 cells. Transitions from therapy-susceptible to therapy-resistant states are well detected by the model, which will prove useful for identifying mutant subtypes in tumors and leading to more optimal therapies for cancer treatment. The tHMM may further be used for drug screening as a single cell, rather than population-based, means of quantifying the potency of novel therapies in eliminating all subpopulations within a tumor. The pipeline will provide researchers and clinicians with an improved classification of heterogeneity among cells, or any other tree-structured data, and provide information about latent changes in cellular identity.
+4. The tHMM can help with uncovering the underlying heterogeneity using single-cell lineage data using only experimental obervations.
+
+Additional findings and how this fits to existing literature:
+
+1. tHMM shows high sensitivity for under-represented sub-populations.
+
+2. tHMM has proved to be more accurate than K-means clustering which shows the importance of accounting for history of cells and their relations.
+
+3. tHMM properly handles time-censorship using survivoral function of distributions, which is barely the case in the literature.
+
+4. Insightful findings from lapatinib treatment fitting, and comparing to some literature about effects of lapatinib on breast cancer cells:
```

Comment:
I'd separate #4/5 into their own paragraph about what we learn about these drugs.